home/forms/domain_model.py

```diff
@@ -0,0 +1,103 @@
+from typing import NamedTuple
+
+
+class Domain(NamedTuple):
+    value: str
```

Comment:
```suggestion
    urn: str
```
small suggestion just to make the code a little easier to parse (propagated)